src/php/ext/grpc/hhvm/LICENSE

```diff
@@ -0,0 +1,186 @@
+
```

Comment:
Should I just remove this file since there is already a LICENSE in the php directory?